docs/strategy.md

```diff
@@ -172,6 +173,28 @@ Further readings:
 - [OPA UA explained in 1 minute (movie)](https://www.youtube.com/watch?v=-tDGzwsBokY)
 - [Industrial Internet Vocabulary](https://www.iiconsortium.org/vocab/index.htm)
 
+## <a id="api-security"></a>API Security
+Equinor APIs should be crafted with high emphasis on security. Code quality is a key foundation for API security. Like other software, APIs should be built by following recognized practices for software development. Examples of such practices includes code review, test driven development (TDD), automated testing, technical debt management & refactoring, etc.
+
+### Security tools and practices
+Development teams should include practices and tools addressing security issues in their development process. Automatic vulnerability scanning and keeping dependencies updated accordingly, is strongly recommended. Teams should stay updated on [OWASP API Security Top 10](https://github.com/OWASP/API-Security), and do regular evaluation on how these security risks are mitigated.
```

Comment:
```suggestion
Development teams should include practices and tools addressing security issues in their development process. Scanning for vulnerabilities in open source dependencies is mandatory. Teams should stay updated on [OWASP API Security Top 10](https://github.com/OWASP/API-Security), and do regular evaluation on how these security risks are mitigated.
```

Reply:
I would say both: 
Automatic vulnerability scanning and keeping dependencies update accordingly, is strongly recommended. Scanning for vulnerabilities in open source dependencies is mandatory.
Your revision is missing the keep updated part.

Reply:
I agree. I left out the keeping updated part, I kind of took that as granted and implicit. I guess we can never be to specific :)